src/nunit.analyzers.tests/nunit.analyzers.tests.csproj

```diff
@@ -14,14 +14,22 @@
     <PackageReference Include="Microsoft.Composition" Version="1.0.31" />
     <PackageReference Include="NUnit" Version="3.12" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
+    <PackageReference Include="StyleCop.Analyzers.Unstable" Version="1.2.0.205">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
-
+  
   <ItemGroup>
     <ProjectReference Include="..\nunit.analyzers\nunit.analyzers.csproj" />
   </ItemGroup>
 
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\..\stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+
```

Comment:
Not really needed as you have added it as a solution item. But I don't mind if you also want it here.

Reply:
Scratch that. It is needed here for StyleCop to pick it up.

Reply:
Yep :(
I've added it as solution item, because I think it is convenient when stylecop.json and .editorconfig are located together, but it makes no functional effect.